lib/logger.js

```diff
@@ -0,0 +1,66 @@
+const logger = (() => {
+  const checkIfLogsEnabled = () => {
+    const search = window?.location?.search
+    const enabled = search && search.includes('debug=true')
+    global.areLogsEnabled = enabled || false
+    return global.areLogsEnabled
+  }
```

Comment:
I'd recommend using an env variable instead of a global for enabling logging [LOGGING_LEVEL="DEBUG"], [LOGGING_LEVEL="TRACE"], [LOGGING_LEVEL="INFO"], [LOGGING_LEVEL="WARN"], [LOGGING_LEVEL="ERROR"]

By default log warn and higher for dev/prod and only log more if the env variable is set to a lower elevation

Reply:
I like this logging level suggestion better than mine. :)

Reply:
Done!